Applications/JupyterHub.qmd

```diff
@@ -1,18 +1,8 @@
-
-Registered Copernicus Data Space Users have access to JupyterLab and Jupyter Notebooks free 
-of charge at a limited capacity of resources beneath.
-
-## Jupyter Notebook and JupyterLab
-
-Jupyter Notebooks serve as an accessible entry point for both non-programmers and developers who want to quickly prototype their EO data processing. They provide a user-friendly interface, integrating well with Python SDK services like Sentinel Hub and OpenEO. Each Notebook has direct access to the EO Data repository, and example notebooks are available to facilitate data interaction. To handle tool dependencies and library versions, multiple notebook kernels are provided and regularly updated with the provided samples. This eliminates the need for users to handle dependency installations, enabling immediate prototyping without technical obstacles.
-
-
-JupyterLab is an advanced interactive development environment (IDE) that offers a flexible and feature-rich interface for working with Jupyter notebooks, code, and data. It surpasses the traditional Jupyter Notebook interface, providing a modern and comprehensive user experience. JupyterLab allows users to organize their workspaces using a flexible layout system with panels, views, and tabs for various activities. It supports various document formats, including Jupyter notebooks, text files, code files, and markdown files. With its modular and extensible architecture, JupyterLab enables customization through extensions, additional functionalities, and integration with external tools. It enhances the user experience with features like a file browser, command palette, debugger, and console, making it a versatile tool for interactive data exploration, analysis, and scientific computing.
+JupyterLab is an advanced interactive development environment (IDE) that offers a flexible and feature-rich interface for working with notebooks, code, and data. It surpasses the Classic Jupyter Notebook Interface, providing a modern and comprehensive user experience. JupyterLab allows users to organize their workspaces using a flexible layout system with panels, views, and tabs for various activities. It supports various document formats, including notebooks, text files, code files, and markdown files. With its modular and extensible architecture, JupyterLab enables customization through extensions, additional functionalities, and integration with external tools. It enhances the user experience with features like a file browser, command palette, debugger, and console, making it a versatile tool for interactive data exploration, analysis, and scientific computing.
```

Comment:
Side-note: consider using an authoring style where you start each sentence on a new line. This will still be rendered as a single paragraph, as markdown requires at least 2 newlines to separate paragraphs.

Having your text chunked over multiple lines makes it a lot easier to do (github based) review and do long term maintenance.

e.g. also see https://sembr.org/

Reply:
> It surpasses the Classic Jupyter Notebook Interface, providing a modern and comprehensive user experience.

 I would not mention the classic webapp

Reply:
noted